ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/IndexPartyManagementService.scala

```diff
@@ -16,4 +18,6 @@ trait IndexPartyManagementService {
   def getParticipantId(): Future[ParticipantId]
 
   def listParties(): Future[List[PartyDetails]]
+
+  def partyEntries(): Source[PartyEntry, NotUsed]
```

Comment:
do we not want the ability to lookup a specific party allocation response as well, for efficiency's sake, especially with the polling to find match?

Reply:
See previous comment.